src/Umbraco.Web/Trees/MacrosTreeController.cs

```diff
@@ -19,6 +20,14 @@ namespace Umbraco.Web.Trees
     public class 
         MacrosTreeController : TreeController
     {
+        protected override TreeNode CreateRootNode(FormDataCollection queryStrings)
```

Comment:
I'm not really sure if this was supposed to be a part of this pull request?

Reply:
Moved to https://github.com/umbraco/Umbraco-CMS/pull/3398.